rust/gui-client/docs/intended_behavior.md

```diff
@@ -40,13 +40,13 @@ Best performed on a clean VM
 1. Sign out of Firezone without quitting
 1. Check the IP again, expect your own IP (Checks for regressions in https://github.com/firezone/firezone/pull/5828)
 1. Export the logs
-1. Expect the zip file to start with "firezone_logs_"
+1. Expect the zip file to start with "firezone*logs*"
```

Comment:
```suggestion
1. Expect the zip file to start with `firezone_logs_`
```
Prettifier kicked in again?

Reply:
Yeah its config changed in another PR so I guess it ran against all files again.